benchexec/test_benchexec_integration.py

```diff
@@ -27,11 +27,15 @@
 import tempfile
 import unittest
 import zipfile
+
+from xml.etree import ElementTree
+
 sys.dont_write_bytecode = True # prevent creation of .pyc files
 
 here = os.path.dirname(__file__)
 base_dir = os.path.join(here, '..')
 bin_dir = os.path.join(base_dir, 'bin')
+tmp_dir = os.path.join(here, 'test_integration')
```

Comment:
Can a real temp dir be used instead of a subdirectory of the source directory? It is better not to write files into the source directory.